src/WebJobs.Script/Binding/Http/HttpBinding.cs

```diff
@@ -14,6 +14,8 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.WebApiCompatShim;
 using Microsoft.Azure.WebJobs.Script.Description;
+using Microsoft.Azure.WebJobs.Script.Grpc.Messages;
```

Comment:
Do you need reference to Microsoft.Azure.WebJobs.Script.Grpc.Messages? It is best to confine reference to Microsoft.Azure.WebJobs.Script.Grpc.Messages to Rpc layer to make it easier to switch Rpc implementation if needed,